package.json

```diff
@@ -126,13 +126,27 @@
         "deno.enable": {
           "type": "boolean",
           "default": false,
-          "markdownDescription": "Controls if the Deno Language Server is enabled. When enabled, the extension will disable the built-in VSCode JavaScript and TypeScript language services, and will use the Deno Language Server (`deno lsp`) instead.\n\n**Not recommended to be enabled globally.**",
+          "markdownDescription": "Controls if the Deno Language Server is enabled. When enabled, the extension will disable the built-in VSCode JavaScript and TypeScript language services, and will use the Deno Language Server instead.\n\n**Not recommended to be enabled globally.**",
           "scope": "resource",
           "examples": [
             true,
             false
           ]
         },
+        "deno.enablePathsOnly": {
```

Comment:
I'm not sure about having "only" in this name as providing this setting kind of implies "only". Some other ideas in no particular order: `deno.enabledPaths` (it seems this one is used elsewhere in the code), `deno.enablePaths`, `deno.paths.enable`, `deno.include`, `deno.paths.include`?

Reply:
I wanted to avoid confusion thinking that you needed to do `deno.enable` plus whatever paths... though I guess there is documentation to clarify that.